recipes/fastpfor/all/patches/cci.20220205-0001-fix-cmake.patch

```diff
@@ -0,0 +1,72 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 5520747..dfccbc1 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -94,6 +94,10 @@ elseif(WIN32)
+     if(NOT MSVC12)
+          message(STATUS "On Windows, only MSVC version 12 is supported!")
+     endif()
++    set (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /arch:AVX")
++    set (CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS_DEBUG} /arch:AVX")
++    set (CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} /arch:AVX")
++    set (CMAKE_C_FLAGS_DEBUG   "${CMAKE_C_FLAGS_DEBUG} /arch:AVX")
```

Comment:
I think this could be included less invasive way (e.g. via `add_definitions` in cmake wrapper)
but I wonder why is it needed?
this will make binaries unable to run on old processors (with no AVX2)